src/wazuh_modules/vulnerability_scanner/src/databaseFeedManager/updateCVEDescription.hpp

```diff
@@ -124,25 +124,30 @@ class UpdateCVEDescription final
 
             VulnDescFBReferenceStr = VulnDescFBReferenceStr.substr(0, VulnDescFBReferenceStr.size() - 2);
 
-            flatbuffers::FlatBufferBuilder builder;
+            if (!(VulnDescFBClassificationStr.empty() || VulnDescFBDescriptionStr.empty() ||
```

Comment:
The `metrics` array isn't mandatory but `descriptions` is.
I wouldn't discard the whole entry.

Reply:
Changed